### PR TITLE
Fix: Dashboard Responsive Web Develop

### PR DESCRIPTION
**Pull request by juntaepark**

### CODE REVIEW

File: packages/web/src/components/dashboard/volume-chart-graph/VolumeChartGraph.tsx [hunk 1]
```diff
@@ -10,25 +11,31 @@ export interface VolumeChartGraphProps {
...
-          datas={datas}
-        />
+        <div className="graph-wrap" ref={componentRef}>
+          <BarGraph
+            className="graph"
+            width={size.width}
+            height={size.height}
+            color={theme.color.background04Hover}
+            hoverColor={theme.color.point}
+            strokeWidth={size.width * 0.022}
```
Comment (jinoosss) [inline packages/web/src/components/dashboard/volume-chart-graph/VolumeChartGraph.tsx:29]: That ratio is something we can discuss at a later date.

File: packages/web/src/hooks/common/use-component-size.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        };
+      setSize({ width, height });
+    };
+
+    handleResize();
+
+    window.addEventListener("resize", handleResize);
+
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
```
Comment (jinoosss) [inline packages/web/src/hooks/common/use-component-size.tsx:27]: It would be better if we could detect changes in the state of the `componentRef`.

**Review by jinoosss — APPROVED** [review]
Approve.
I left a few suggestions.

File: packages/web/src/components/dashboard/volume-chart-graph/VolumeChartGraph.tsx (hunk 1 above)
Comment (juntaepark) [reply packages/web/src/components/dashboard/volume-chart-graph/VolumeChartGraph.tsx:29]: i think so too.

File: packages/web/src/hooks/common/use-component-size.tsx (hunk 2 above)
Comment (juntaepark) [reply packages/web/src/hooks/common/use-component-size.tsx]: Edited because I thought it was a good comment.

**Review by jinoosss — APPROVED** [review]

**Pull request merged**